doc/api/console.markdown

```diff
@@ -109,6 +109,45 @@ console.assert(false, 'Whoops %s', 'didn\'t work');
   // AssertionError: Whoops didn't work
 ```
 
+It is important to note that the `console.assert()` method in Node.js is
+implemented differently than the `console.assert()` method available in
+browsers. Specifically, in browsers, calling `console.assert()` with a falsy
```

Comment:
perhapse a link to https://developer.mozilla.org/en-US/docs/Web/API/console/assert